presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -8446,11 +8446,11 @@ public void testSwitchReturnsNull()
     public void testAndInFilter()
     {
         assertQuery(
-                "SELECT count() from (select * from orders where orderkey < random(10)) where ((orderkey > 100 and custkey > 100) or (orderkey > 200 and custkey < 200))",
+                "SELECT count() from (select * from orders where orderkey < random(10) + 1) where ((orderkey > 100 and custkey > 100) or (orderkey > 200 and custkey < 200))",
```

Comment:
@kaikalur why were these tests using random(10) instead of an explicit value for the filter (or just a limit)?

Reply:
Good point. We can make it a between 0 and 10 or something.